cli/src/test/java/com/devonfw/tools/ide/environment/EnvironmentVariablesPropertiesFileTest.java

```diff
@@ -37,4 +42,96 @@ public void testLoad() {
     assertThat(variables.getVariables()).hasSize(4);
   }
 
+  @Test
+  void testSave() {
+
+    // arrange
+    List<String> linesToWrite = new ArrayList<>();
+    linesToWrite.add("# first comment");
+    linesToWrite.add("# second comment");
+    linesToWrite.add("var0=0");
+    linesToWrite.add("var1=1");
+    linesToWrite.add("var2=2");
+    linesToWrite.add("export var3=3");
+    linesToWrite.add("export var4=4");
+    linesToWrite.add("export var5=5");
+    linesToWrite.add("export var6=6");
+    linesToWrite.add("# third comment");
+    linesToWrite.add("var7=7");
+    linesToWrite.add("var8=8");
+    linesToWrite.add("# 4th comment");
+    linesToWrite.add("# 5th comment");
+    linesToWrite.add("var9=9");
+
+    Path propertiesFilePath = Path.of("src/test/resources/com/devonfw/tools/ide/env/var/ide.properties");
```

Comment:
never write to `src` in tests. This is causing changes that will then appear in git and may be added and committed accidentally. Mutable testdata shall be in target or temp-folder.

Reply:
I tried using _context.getFileAccess().createTempDir()_ but since this does not seem to work when git-actions runs the tests I will be using the target dir.